src/Core/Managed/Shared/Extensibility/Implementation/JsonSerializer.cs

```diff
@@ -54,6 +54,51 @@ public static byte[] Serialize(IEnumerable<ITelemetry> telemetryItems, bool comp
         }
 
         /// <summary>
+        /// Converts serialized telemetry items to a byte array.
+        /// </summary>
+        /// <param name="telemetryItems">Serialized telemetry items.</param>
+        /// <param name="compress">Should serialization also perform compression.</param>
+        /// <returns>The compressed and serialized telemetry items.</returns>
+        [SuppressMessage("Microsoft.Usage", "CA2202:Do not dispose objects multiple times", Justification = "Disposing a MemoryStream multiple times is harmless.")]
+        public static byte[] ConvertToByteArray(string telemetryItems, bool compress = true)
+        {
```

Comment:
Do we need ConvertToByArray - how is it different from Serialize method we already have?